static/js/typeahead_helper.js

```diff
@@ -379,10 +379,3 @@ export function sort_streams(matches, query) {
 
     return name_results.matches.concat(desc_results.matches.concat(desc_results.rest));
 }
-
-export function sort_recipientbox_typeahead(query, matches, current_stream) {
-    // input_text may be one or more pm recipients
-    const cleaned = get_cleaned_pm_recipients(query);
-    query = cleaned[cleaned.length - 1];
-    return sort_recipients(matches, query, current_stream);
-}
```

Comment:
I wanted to explain this change in more detail. I wanted to add the details about how possible ways handling invalid input in pill accepting input field are handled by our pill code(`user_pill.js`,  `stream_pill.js`, `user_group_pill.js`). 
I have explained this using `user_pill.js` as code all three files is in similar fashion. So if someone pastes
```quote
user8@zulipdev.com, , user7@zulipdev.com, user6@zulipdev.com,
```
then because we filter all invalid users so `, ,` does not get considered when we create or fetch input box items. Infact it gets removed during the creation of pills on pasting the text.

If someone pastes like input like:
```quote
user8@zulipdev.com, , blahblah, user7@zulipdev.com, user6@zulipdev.com
```
then pills corresponding to correct emails will get created and the `blahblah` will be placed after all the pills and since it is not a valid pill item it is not included whenever we fetch items in the input widget as it only gives valid items.

Like this I have tested all other possible cases of giving such invalid input to these widgets(like manually typing `,` separated recipients) but since we create pills only for valid recipients using
```javascript
    const user = people.get_by_email(email);

    if (!user) {
```
and then include only valid pill items whenever we fetch items from the pill input widget using
```javascript
export function get_user_ids(pill_widget) {
    const items = pill_widget.items();
    let user_ids = items.map((item) => item.user_id);
    user_ids = user_ids.filter(Boolean); // be defensive about undefined users

    return user_ids;
}
```
all such cases of invalid inputs that can affect typeahead sorter are handled. So this can be safely removed.

Reply:
Hmm, when I try pasting a set of email addresses into the compose box in master, it seems to not work, both before and after this change.